src/network_systems/projects/can_transceiver/inc/can_frame_parser.h

```diff
@@ -140,13 +144,11 @@ class BaseFrame
 class Battery final : public BaseFrame
 {
 public:
-    // Valid CanIds that a Battery object can have
-    static constexpr std::array<CanId, 2> BATTERY_IDS       = {CanId::BMS_P_DATA_FRAME_1, CanId::BMS_P_DATA_FRAME_2};
-    static constexpr uint8_t              CAN_BYTE_DLEN_    = 8;
-    static constexpr uint8_t              BYTE_OFF_VOLT     = 0;
-    static constexpr uint8_t              BYTE_OFF_CURR     = 2;
-    static constexpr uint8_t              BYTE_OFF_MAX_VOLT = 4;
-    static constexpr uint8_t              BYTE_OFF_MIN_VOLT = 6;
+    // Valid CanIds that a Battery object can have. There used to be two, but now there is only one.
+    static constexpr std::array<CanId, 1> BATTERY_IDS    = {CanId::BMS_DATA_FRAME};
```

Comment:
Since we only have one type of CAN frame for the battery, do we still need to use the `BATTERY_IDS` array of one element? there should be no need to verify the CAN ID of the message now. 

Please remove the `BATTERY_IDS` array, update `publishBattery` in `can_transciever_ros_intf.cpp` to not go through the for loop,  and update the publisher to publish `HelperBattery.msg`. Keep the original `Batteries.msg` for now.